protocol-units/execution/opt-executor/src/executor/mod.rs

```diff
@@ -46,7 +46,7 @@ impl Executor {
 		self.transactions_in_flight
 			.fetch_update(Ordering::Relaxed, Ordering::Relaxed, |current| {
 				info!(
-					target: "movement_timing",
+					target: "movement_telemetry",
 					count,
 					current,
 					"decrementing_transactions_in_flight",
```

Comment:
Also this one looks like a metric would be better (I haven't converted it to OpenTelemetry yet).